.travis.yml

```diff
@@ -28,23 +28,12 @@ matrix:
           language: generic
           osx_image: xcode7.3
           env: SYSTEM_TEST=1
-        # test on osx yosemite
-        - os: osx
-          sudo: required
-          language: generic
-          osx_image: xcode7.1
-          env: SYSTEM_TEST=1
-        # test with default python on trusty and precise
+        # test with default python on trusty
         - os: linux
           sudo: required
           dist: trusty
           language: generic
           env: SYSTEM_TEST=1
-        - os: linux
```

Comment:
@jvehent how do you feel about removing yosemite and precise testing on CI?

Reply:
XCode 7.1 no longer appears in the Travis docs: https://docs.travis-ci.com/user/reference/osx/#Xcode-version

I dropped precise because Ubuntu 12.04 is EOL